src/nrnpython/CMakeLists.txt

```diff
@@ -91,7 +91,8 @@ install(
   PATTERN *.py
   PATTERN *.so
   PATTERN *.dylib
-  PATTERN *.dat)
+  PATTERN *.dat
+  PATTERN *.json)
```

Comment:
I'm not really sure what effect this has. On my machine,
```
~/neuron/cv3new/build$ ls share/lib/python/neuron
rxd
```
so I guess I don't know why any of the
```
install(
  DIRECTORY ${PROJECT_BINARY_DIR}/share/lib/python
```
is there.